### PR TITLE
Rams api

### PR DESCRIPTION
**Pull request by simhmiller**
Adding following to docs:
  - project duplication
  - project duplication process
  - personnel assignments
  - latest reviews
  - project archiving
  - version pdf included history context (update)
  - new event subscriptions and webhooks:
      - project_archived 
      - project_unarchived
      - personnel_assignment_created 
      - personnel_assignment_updated 
      - personnel_assignment_deleted

### CODE REVIEW

File: source/includes/_event_notifications.md [hunk 1]
```diff
@@ -69,6 +75,9 @@ The body of an event subscription will vary depending on the type of event being
             "id": "19",
             "type": "project"
           }
+        },
+        "message": {
+          "data": null
```
Comment (kknevitt) [inline source/includes/_event_notifications.md:80]: interesting, do we ever expose that in Zapier? if not maybe we should remove it from the API history serializer?

File: source/includes/_event_notifications.md [hunk 2]
```diff
@@ -137,6 +147,201 @@ The body of an event subscription will vary depending on the type of event being
...
+   "meta": {
+      "event_type": "project_archived"
+   }
+}
+```
+### Data:
+
+  - The attributes of the project, along with IDs of the project's user (author) and fields.
+
+## Project Archived
```
Comment (kknevitt) [inline source/includes/_event_notifications.md:192]: Should this be:
```suggestion
## Project Unarchived
```

File: source/includes/_event_subscriptions.md [hunk 3]
```diff
@@ -83,6 +83,11 @@ Events currently supported are:
 
 ### RAMS
 - `version_pdf_created` - this is fired after we have generated and stored a project version PDF, which is now ready to be downloaded. Please note that the event is scoped to look for subscriptions which were created by the same division that the project belongs to.
+- `project_archived` - this is fired after a project is archived (via the HandsHQ app or via the API).
+- `project_unarchived` - this is fired after a project is unarchived (via the HandsHQ app or via the API).
+- `personnel_assignment_created` - this is fired after a personnel has been added to a project (via the HandsHQ app or via the API).
+- `personnel_assignment_updated` - this is fired after a personnel role has been updated on a project (via the HandsHQ app or via the API).
```
Comment (kknevitt) [inline source/includes/_event_subscriptions.md:89]: I don't think we should make this specific to role, it should just be about the details of the assignment of a personnel to a project - though I think it's fine as an example of an update. I know that's the only thing that can change atm but that can reasonably change in the future

File: source/includes/_latest_reviews.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
+# Latest Reviews
+
+A version of a project can be reviewed by the personnel that have been assigned to that project.
+
+A personnel can review multiple versions of a project, and you can use this endpoint to access their review for the latest version of the project.
+
+Reviews will only be found once personnel have been added to the project and reviews have been requested for this version of the project. If you add a personnel to a project after reviews have been requested for that version, a review will automatically be created.
+
+A review can have the state of requested, accepted or rejected. Reviews that have been accepted will have a corresponding signature.
```
Comment (kknevitt) [inline source/includes/_latest_reviews.md:9]: Nice summary 👍

File: source/includes/_personnel_assignments.md [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+    "user_email": "maddox@daystrom.com",
+    "personnel_assignment": {
+      "role_id": "123"
+    }
+  }
+```
+
+This endpoint allows you to add a personnel to a project within the division that is registered with the API token you provide. You have the option to determine the role of the personnel on the project, by providing the id of the role. This role must have already been assigned to the personnel.
+
+Once a personnel is added to a project, if reviews have been requested for that project, a review will automatically be generated for that personnel. A user email must be provided along with this request, and this email will be used to identify the requester of the review.
```
Comment (kknevitt) [inline source/includes/_personnel_assignments.md:150]: 👍

File: source/includes/_version_pdfs.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
+# Version PDFs
+
+A version pdf is a pdf representation of the current version of the project. As a project changes over time, different versions are generated, and these versions can have pdf representations generated.
```
Comment (kknevitt) [inline source/includes/_version_pdfs.md:3]: 👍

**Review by kknevitt — COMMENTED** [review]
I know it's in draft but figured I'd get a headstart, left some suggestions for changes

File: source/includes/_event_notifications.md (hunk 1 above)
Comment (simhmiller) [reply source/includes/_event_notifications.md:80]: Hi @kknevitt. We don't - but can't guarantee no one with a direct integration to the api isn't using it. Happy to remove it?

File: source/includes/_event_notifications.md (hunk 2 above)
Comment (simhmiller) [reply source/includes/_event_notifications.md:192]: Yes 🙌

**Comment by simhmiller** [comment]
Thanks @kknevitt

Have made those changes. Will also remove message from history serializer if you're happy?

We didn't confirm whether to update latest reviews to latest document reviews in our documentation / on zapier. 
In terms of the app, do you think the url /latest_reviews is clear enough or would /latest_document_reviews be a better choice?

**Pull request merged**